test/lit.cfg

```diff
@@ -1702,7 +1703,7 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         config.swiftc, '-emit-pcm',
         '-target', config.variant_triple])
     config.target_clang = ' '.join([
-        'clang++',
+        '{}/clang++'.format(tools_directory),
```

Comment:
This was using the system clang, which is wrong. I tried using the Swift-forked toolchain clang first and that didn't work, so ended up with this NDK clang. My guess is that it is the only one that finds the C++ headers because it is located in the NDK next to them, unlike the other clangs tried.

Reply:
Given that the `-tools-directory` is supposed to take precedence for selecting the tools, using the `clang++` from the tools directory doesn't seem too terribly wrong.